src/applications/ivc-champva/10-7959f-1/helpers/maskSSN.jsx

```diff
@@ -0,0 +1,57 @@
+import { formatSSN } from 'platform/utilities/ui';
+import React, { useState, useEffect, useRef } from 'react';
+import vaTextInputFieldMapping from 'platform/forms-system/src/js/web-component-fields/vaTextInputFieldMapping';
+import { VaTextInput } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+function maskSSN(ssnString = '') {
+  const strippedSSN = ssnString.replace(/[- ]/g, '');
+  const maskedSSN = strippedSSN.replace(/^\d{1,5}/, digit =>
+    digit.replace(/\d/g, '●'),
+  );
+  return formatSSN(maskedSSN);
+}
+
+/**
+ * @param {WebComponentFieldProps} props */
+export default function HandlePrefilledSSN(fieldProps) {
+  const props = vaTextInputFieldMapping(fieldProps);
+  const [val, setVal] = useState(props.value);
+  const [displayVal, setDisplayVal] = useState(props.value);
+  const vaTextInput = useRef();
+
+  useEffect(() => {
+    setVal(maskSSN(val));
+    setDisplayVal(maskSSN(val));
+  }, []);
+  const handleChange = event => {
+    const { value } = event.target;
+    let strippedSSN;
+    if (value) {
+      strippedSSN = value.replace(/[- ]/g, '');
+    }
+
+    setVal(value);
+    setDisplayVal(value);
```

Comment:
Might need to track a boolean that indicates if the user has cleared the field at least once, then we can show all characters like we do here - otherwise we should only ever show the sanitized SSN

Reply:
I got this working just as you said. Added a bool and then tracked it to be sure that the field had been cleared.